packages/desktop-client/src/components/reports/ReportLegend.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { theme, styles } from '../../style';
+import Text from '../common/Text';
+import View from '../common/View';
+
+type ReportLegendProps = {
+  legend: Array<{ name: string; color: string }>;
+  groupBy: string;
+};
+
```

Comment:
Same here (no code changes). Pulled it out of reportSummary to separate as it's own element and added typescript.